packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -294,3 +265,90 @@ export async function checkForAssertions({
     )
   }
 }
+
+export async function fundUsdc({
+  address, // wallet address where funding is required
+  provider,
+  amount,
+  networkType,
+  sourceWallet
+}: {
+  address: string
+  provider: Provider
+  amount: BigNumber
+  sourceWallet: Wallet
+  networkType: NetworkType
+}) {
+  console.log('Funding USDC to user wallet...')
+  const usdcContractAddress =
+    networkType === 'parentChain'
+      ? CommonAddress.Sepolia.USDC
+      : CommonAddress.ArbitrumSepolia.USDC
+
+  const contract = new ERC20__factory().connect(sourceWallet.connect(provider))
+  const token = contract.attach(usdcContractAddress)
+  await token.deployed()
+  const tx = await token.transfer(address, amount)
+  await tx.wait()
+}
+
+export async function fundEth({
+  address, // wallet address where funding is required
+  provider,
+  sourceWallet, // source wallet that will fund the `address`,
+  amount = utils.parseEther('2')
+}: {
+  address: string
+  provider: Provider
+  sourceWallet: Wallet
+  amount?: BigNumber
+}) {
+  console.log(`Funding ETH to user wallet ${address}...`)
```

Comment:
Needs `networkType` in this function. We should add it again.

Reply:
I would probably move the log outside then, it feels weird to pass a parameter only to log it